fatiando/gravmag/transform.py

```diff
@@ -299,3 +397,31 @@ def _fftfreqs(x, y, shape, padshape):
     dy = (y.max() - y.min())/(ny - 1)
     fy = 2*numpy.pi*numpy.fft.fftfreq(padshape[1], dy)
     return numpy.meshgrid(fy, fx)[::-1]
+
+
+def thdr(x, y, data, shape, method = 'fd'):
+    """
+    Total Horizontal Derivative
+
+    A useful thing used in many tilt angle filters:
+
+    sqrt( \\frac{df}{dx}^2 + \\frac{df}{dy}^2 )
```

Comment:
You need to put the line above in a `.. math::` block for the it to render in the docs [like this](https://github.com/fatiando/fatiando/blob/master/fatiando/gravmag/transform.py#L90).

Also, if you declare the docstring as a "raw string" then you don't have to write `\\` every time. Do this by opening the string with `r"""`.